### PR TITLE
Make autosklearn/util modules PEP8 compliant

### PR DESCRIPTION
**Pull request by gui-miotto**

### CODE REVIEW

File: ci_scripts/flake8_full.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+flake8 --max-line-length=100 --show-source autosklearn/util
```
Comment (gui-miotto) [inline ci_scripts/flake8_full.sh:1]: I added this new script. It should bit by bit substitute the flake8_diff.sh

**Comment by gui-miotto** [comment]
@mfeurer : I think autosklearn/util is good to go.

File: autosklearn/util/__init__.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # -*- encoding: utf-8 -*-
-from .common import *
```
Comment (mfeurer) [inline autosklearn/util/__init__.py:2]: Would it be possible to change/remove these imports?

Comment (gui-miotto) [reply autosklearn/util/__init__.py:2]: yes

Comment (gui-miotto) [reply autosklearn/util/__init__.py:2]: No, wait. Just to clarify. Do you mean change/remove without modifying the way those functions are imported by the rest of the package?

Comment (mfeurer) [reply autosklearn/util/__init__.py:2]: No, I meant removing those imports and changing the way other modules import the utils.

Comment (gui-miotto) [reply autosklearn/util/__init__.py:2]: Alright then!

**Pull request merged**